handlers/node_api.py

```diff
@@ -176,6 +176,12 @@ def get_transaction(
     if id is None:
         raise ApiError("invalid_parameters")
     response = node_api.get_transaction(id)
+    # It does not make sense to show in the explorer thousands of children.
+    # The full node will continue returning the correct data but in the explorer
+    # service we will truncate it and return only the latest 100 to show in the UI
+    # (it might even be more than we should, maybe we should paginate in the future)
+    if 'meta' in response and 'children' in response['meta']:
+        response['meta']['children'] = response['meta']['children'][-100:]
```

Comment:
Personally I think we can be stricter with this.
We can just cap at 20 instead.

Reply:
I agree with you, 20 should be more than enough. But we are currently showing everything, so I prefer keeping it as a "big" number for now, and we can reduce it in the future if we prefer.